tests/test_icdar_format.py

```diff
@@ -323,3 +323,38 @@ def test_can_save_and_load_captions_with_quotes(self):
             self._test_save_and_load(expected_dataset,
                 partial(IcdarWordRecognitionConverter.convert, save_images=True),
                 test_dir, 'icdar_word_recognition')
+
+    @mark_requirement(Requirements.DATUM_GENERAL_REQ)
+    def test_can_save_and_load_segm_wo_color_attribute(self):
+        source_dataset = Dataset.from_iterable([
+            DatasetItem(id='1', subset='train',
+                image=np.ones((10, 15, 3)), annotations=[
+                    Mask(image=np.array([[0, 0, 0, 1, 1]]), group=1,
+                        attributes={'index': 1, 'text': 'j', 'center': '0 3',
+                            'color': '0 128 0'}),
```

Comment:
Having colors defined here contradicts the test purpose.

Reply:
I added one defined color for testing functionality according [this comment](https://github.com/openvinotoolkit/datumaro/pull/556#discussion_r751974896)